configs/components/openssl-1.1.1.rb

```diff
@@ -8,6 +8,10 @@
   # ENVIRONMENT, FLAGS, TARGETS
   #############################
 
+  # make sure openssl-lib compiles first, as we it only installs versioned libs and hopefully will not cause problems
+  # the other way around caused: # error "Inconsistency between crypto.h and cryptlib.c"
+  pkg.build_requires 'openssl-lib' if platform.is_linux?
```

Comment:
This is trying to install an 'openssl-lib' package via the package manager - seems like the intention is to have this be dependent on the earlier openssl-lib build?

Reply:
correct